Source/AwsCommonRuntimeKit/http/HttpMonitoringOptions.swift

```diff
@@ -2,28 +2,25 @@
 //  SPDX-License-Identifier: Apache-2.0.
 import AwsCHttp
 
-public class HttpMonitoringOptions {
+public class HttpMonitoringOptions: CStruct {
     /// Minimum amount of throughput, in bytes per second, for a connection to be considered healthy
-    public let minThroughputBytesPerSecond: Int
+    public var minThroughputBytesPerSecond: UInt64
```

Comment:
I genuinely don't know if we want members of options structs/classes to all be `let` or `var`, but we should probably be consistent

it's pretty inconsistent right now

Reply:
I think It should be var, as we make c structs on the fly, there is no point in making it let and forcing users to create new objects if they want to modify something.

Reply:
- Changing all to var
- Callbacks are a let because of special handling required with retain/release.